packages/ui/Changeable/tests/Changeable-specs.js

```diff
@@ -81,13 +81,13 @@ describe('Changeable', () => {
 			expect(actual).toBe(expected);
 		});
 
-		test('should use value prop when value prop is null', () => {
+		test('should use defaultValue prop when value prop is null', () => {
```

Comment:
Just leaving a note that we need to address this later as `null` can be a valid value.